CountryPicker/CountryPicker/CountryManager.swift

```diff
@@ -43,8 +54,21 @@ open class CountryManager {
         return Country(countryCode: countryCode)
     }
     
+    
+    internal var lastCountrySelected: Country?
+    
+    /// Default country filter option
+    internal let defaultFilter: CountryFilterOptions = .countryName
+    
+    /// Exposed country filter options and should be configured by user
+    ///
+    /// - Note: By default, countries can be filtered by there country names
+    internal var filters: [CountryFilterOptions] = [.countryName]
```

Comment:
May be we can use `Set` as better representable data structure here. so only we need to `filter.instert` will take care of duplication

Reply:
Thanks for pointing that out. That will definitely solve that duplication issue.